net/netatalk4/Portfile

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           meson 1.0
+
+name                netatalk4
+conflicts           netatalk
+
+github.setup        Netatalk netatalk netatalk-4-1-2
+github.tarball_from archive
+revision            0
+
+checksums           rmd160  b6953e69144d6542c1b426015635d58f695cdba9 \
+                    sha256  012f2a32c4904f4010c1b924e5c01cf16dfc628cb32965be54479721f30aa30d \
+                    size    1327961
+
+categories          net
+license             GPL-2+
+maintainers         {openmaintainer github:trodemaster}
+
+description         Netatalk is a freely-available Open Source AFP fileserver.
+long_description    Netatalk is a freely-available Open Source AFP fileserver. \
+                    It allows Unix-like operating systems to serve as file \
+                    servers for Macintosh computers.
+
+homepage            https://netatalk.io
+
+depends_build       path:bin/pkg-config:pkgconfig
+
+depends_lib         port:db48 \
+                    port:libevent \
+                    port:libgcrypt
+
+platform darwin {
+    if {${os.major} < 10} {
+         variant appletalk description {Build with AppleTalk support} {
+              configure.args-append -Dwith-appletalk=true
+         }
+    } else {
+         variant appletalk description {Build with AppleTalk support (removed)} {
+             error "AppleTalk support has been removed from macOS versions greater than 10.5.x ."
+         }
+    }
+}
+
+configure.args-append -Dwith-init-style=none \
+                      -Dwith-pam-config-path=${prefix}/etc/pam.d \
+                      -Dwith-lockfile-path=${prefix}/var/run
+
+startupitem.create  yes
+startupitem.executable  ${prefix}/sbin/netatalk -d -F ${prefix}/etc/afp.conf
+startupitem.logfile ${prefix}/var/log/${name}.log
+startupitem.logfile.stderr ${prefix}/var/log/${name}-stderr.log
+startupitem.logevents yes
+startupitem.debug          yes
+
+livecheck.type      regex
+livecheck.url       https://github.com/Netatalk/netatalk/releases
+livecheck.regex     {Latest release.*?v?(\d+(?:\.\d+)+)}
```

Comment:
livecheck should use the defaults from the `github` PortGroup, just remove all three lines

Reply:
Removed